core/src/main/java/org/geysermc/geyser/registry/type/ItemMappings.java

```diff
@@ -159,10 +161,19 @@ public ItemMapping getMapping(ItemData data) {
     }
 
     @Override
-    public ItemDefinition getDefinition(int bedrockId) {
+    public @Nullable ItemDefinition getDefinition(int bedrockId) {
         return this.itemDefinitions.get(bedrockId);
     }
 
+    public @Nullable ItemDefinition getDefinition(String bedrockIdentifier) {
```

Comment:
sorry - put the annotation on a newline above the method please, just to match the rest of this class

also, use the checkerframework annotations, not javax or lombok

And annotate the other two getMapping methods in this class that I forgot to mention